### ISSUE TITLE
Accessibility: Image with no alt attribute - Exhibit cards

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#419]
**Description**
On each exhibit card, the image that is shown is missing an alt attribute. Investigating into the Appearance section, there does not seem to be a field to add alt text for the Exhibit Thumbnail.

[Homepage report](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=48145876144&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/1.1.1/Check/1)

**Solution**
Option 1: Add a required field for alt text that will display for these images if an image is uploaded. If an image from the exhibit is chosen, does it already have alt text that could be used?
Option 2: If all card images are solely decoration, add alt="" to all images on the cards.

**Comment by elrayle** [comment]
Addressed by PR https://github.com/projectblacklight/spotlight/pull/2660

Closing of this issue is pending:

* merge in Spotlight release-2.x
* new release of Spotlight 2.x series
* update of our Spotlight to the new 2.x release

**Comment by chrisrlc** [comment]
No longer listed as an issue in SiteImprove. Possibly resolved from prior spotlight upgrades. Image thumbnails now show with "alt role='presentation'" tags
